src/NuGet.Clients/NuGet.PackageManagement.VisualStudio/RemoteTypes/IRemoteSourceRepositoryProvider.cs

```diff
@@ -0,0 +1,9 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using NuGet.Protocol.Core.Types;
+
+namespace NuGet.PackageManagement.VisualStudio.RemoteTypes
+{
+    public interface IRemoteSourceRepositoryProvider : ISourceRepositoryProvider { }
```

Comment:
I'm curious why you define this empty interface that just derives from another. 
In general, we encourage brokered service interfaces to be the primary interface for the feature rather than considered only the "remote" version, since we want parity whether your feature is local or remote.